packages/plugin-inline/src/Inline.ts

```diff
@@ -72,14 +72,10 @@ export class Inline<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<
                     const format = inline.modifiers.find(FormatClass);
                     // Apply the attributes of the format we're about to remove to
                     // the inline itself.
-                    const attributes = inline.modifiers.find(Attributes);
-                    const matchingFormatAttributes = format.modifiers.find(Attributes);
-                    if (attributes && matchingFormatAttributes) {
-                        for (const key of matchingFormatAttributes.keys()) {
-                            attributes.set(key, matchingFormatAttributes.get(key));
-                        }
-                    } else if (matchingFormatAttributes) {
-                        inline.modifiers.append(matchingFormatAttributes.clone());
+                    const attributes = inline.modifiers.get(Attributes);
+                    const matchingFormatAttributes = format.modifiers.get(Attributes);
```

Comment:
We don't write on this one, do we ? We just read it, right ? Then it should be a `find`.